combat-trainer.lic

```diff
@@ -189,12 +191,46 @@ class LootProcess
   def dispose_body(game_state)
     return if DRRoom.dead_npcs.length < 1
     game_state['mob_died'] = true
-
-    check_skinning(DRRoom.dead_npcs.first) if @skin
-
+      if @necro
```

Comment:
Where does @necro come from? I don't see it in the parameter list to initialize.